src/common/commonutils/DaemonUtils.c

```diff
@@ -3,6 +3,32 @@
 
 #include "Internal.h"
 
+#define MAX_DAEMON_NAME_LENGTH 256
+
+bool IsValidDaemonName(const char *name) 

```

Comment:
What IS a valid daemon name? Just by looking at the code its no numbers, underscores and dashes? Could you include that as function comments?

Reply:
Correct. Those names cannot be anything. Only these characters.